.rhdh/scripts/install-rhdh-catalog-source.sh

```diff
@@ -0,0 +1,221 @@
+  - mirrors:
+    - ${ICSP_URL}rhdh-rhel9-operator
+    source: registry-proxy.engineering.redhat.com/rh-osbs/rhdh-rhdh-rhel9-operator
+
+  - mirrors:
+    - ${ICSP_URL}rhdh-hub-rhel9
+    source: registry.redhat.io/rhdh/rhdh-hub-rhel9
+  - mirrors:
+    - ${ICSP_URL}rhdh-hub-rhel9
+    source: registry.stage.redhat.io/rhdh/rhdh-hub-rhel9
+  - mirrors:
+    - ${ICSP_URL}rhdh-hub-rhel9
+    source: registry-proxy.engineering.redhat.com/rh-osbs/rhdh-rhdh-hub-rhel9
+
+  ## 2. general repo mappings
+  - mirrors:
+    - ${ICSP_URL_PRE}
+    source: registry.redhat.io
+  - mirrors:
+    - ${ICSP_URL_PRE}
+    source: registry.stage.redhat.io
+  - mirrors:
+    - ${ICSP_URL_PRE}
+    source: registry-proxy.engineering.redhat.com
+
+  ### now add mappings to resolve internal references
+  - mirrors:
+    - registry.redhat.io
+    source: registry.stage.redhat.io
+  - mirrors:
+    - registry.stage.redhat.io
+    source: registry-proxy.engineering.redhat.com
+  - mirrors:
+    - registry.redhat.io
+    source: registry-proxy.engineering.redhat.com
+" > "$TMPDIR/ImageContentSourcePolicy_${ICSP_URL_PRE}.yml" && oc apply -f "$TMPDIR/ImageContentSourcePolicy_${ICSP_URL_PRE}.yml"
+
+echo "[INFO] Using iib from image $UPSTREAM_IIB"
+IIB_IMAGE="${UPSTREAM_IIB}"
+CATALOGSOURCE_NAME="${TO_INSTALL}-${OLM_CHANNEL}"
+DISPLAY_NAME_SUFFIX="${TO_INSTALL}"
+
+# Add CatalogSource for the IIB
+if [ -z "$TO_INSTALL" ]; then
+  IIB_NAME="${UPSTREAM_IIB##*:}"
+  IIB_NAME="${IIB_NAME//_/-}"
+  IIB_NAME="${IIB_NAME//./-}"
+  IIB_NAME="$(echo "$IIB_NAME" | tr '[:upper:]' '[:lower:]')"
+  CATALOGSOURCE_NAME="rhdh-iib-${IIB_NAME}-${OLM_CHANNEL}"
+  DISPLAY_NAME_SUFFIX="${IIB_NAME}"
+fi
+echo "apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: ${CATALOGSOURCE_NAME}
+  namespace: ${NAMESPACE_CATALOGSOURCE}
+spec:
+  sourceType: grpc
+  image: ${IIB_IMAGE}
+  publisher: IIB testing ${DISPLAY_NAME_SUFFIX}
+  displayName: IIB testing catalog ${DISPLAY_NAME_SUFFIX}
+" > $TMPDIR/CatalogSource.yml && oc apply -f $TMPDIR/CatalogSource.yml
+
+if [ -z "$TO_INSTALL" ]; then
+  echo "Done. Now log into the OCP web console as an admin, then go to Operators > OperatorHub, search for Red Hat Developer Hub, and install the Red Hat Developer Hub Operator."
+  exit 0
+fi
+
+# Create OperatorGroup to allow installing all-namespaces operators in $NAMESPACE_SUBSCRIPTION
+echo "Creating OperatorGroup to allow all-namespaces operators to be installed"
+echo "apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: rhdh-operator-group
+  namespace: ${NAMESPACE_SUBSCRIPTION}
+" > $TMPDIR/OperatorGroup.yml && oc apply -f $TMPDIR/OperatorGroup.yml
+
+# Create subscription for operator
+echo "apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: $TO_INSTALL
+  namespace: ${NAMESPACE_SUBSCRIPTION}
+spec:
+  channel: $OLM_CHANNEL
+  installPlanApproval: $INSTALL_PLAN_APPROVAL
```

Comment:
This var isn't defined -- should it be `automatic` ?

Reply:
Good catch. Fixed in [`2479f49` (#184)](https://github.com/janus-idp/operator/pull/184/commits/2479f49e7dd4226e069f35e774c5e71423410381)